reframe/frontend/cli.py

```diff
@@ -405,13 +405,15 @@ def main():
             exec_policy.sched_nodelist = options.nodelist
             exec_policy.sched_exclude_nodelist = options.exclude_nodes
             exec_policy.sched_options = options.job_options
-            runner = Runner(exec_policy, printer)
+            # TODO: Get this later from the user; set to 0 by default (get_checks_failed programmed such that can be get per partition)
+            max_retries = 2
```

Comment:
This could be the reason for the current failure of the async unit tests in my branch. Once I do not have it hardcoded, here it should pass again, I guess.

Reply:
I would say to update with the latest master, because otherwise we cannot be sure. I am quite confident that the old failure is fixed properly.